debezium-core/pom.xml

```diff
@@ -80,6 +80,27 @@
             <artifactId>kafka-connect-avro-converter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
There's a test utility in core that can be used in connector-specific tests to help verify `SourceRecords` are correct w/r/t their schemas and that they are serializable/deserializable via Avro. Thus, this utility has a _test_ dependency on the Avro converter.